blocktx/blocktx_api/blocktx_api.proto

```diff
@@ -100,15 +103,21 @@ message Source {
   string source = 1;
 }
 
+// swagger:model Source
+message MerklePath {
+  string merklePath = 1;
+}
+
 message TransactionAndSource {
   bytes hash = 1;
   string source = 2;
 }
 
 message RegisterTransactionResponse {
   string source = 1;
-  bytes block_hash = 2;
-  uint64 block_height = 3;
+  string merkle_path = 2;
```

Comment:
I think it would be better to add the merkle path at the end as the proto object becomes in compatible with a previous version.

Reply:
same as above